ariadne/utils.py

```diff
@@ -23,3 +25,27 @@ def unwrap_graphql_error(
     if isinstance(error, GraphQLError):
         return unwrap_graphql_error(error.original_error)
     return error
+
+
+def convert_kwargs_snake_case(func):
+    def convert_to_snake_case(d):
+        converted = {}
+        for k, v in d.items():
+            if isinstance(v, dict):
+                v = convert_to_snake_case(v)
+            converted[convert_camel_case_to_snake(k)] = v
+        return converted
+
+    if asyncio.iscoroutinefunction(func):
+
```

Comment:
```suggestion
```

Reply:
I did not want that empty line either, but black enforces it.

Reply:
Like... before decorator? Allright then.